apps/gdallocationinfo.cpp

```diff
@@ -137,6 +137,12 @@ MAIN_START(argc, argv)
         .help(_("Query the (overview_level)th overview (overview_level=1 is "
                 "the 1st overview)."));
 
+    std::string osResampling;
+    argParser.add_argument("-r")
```

Comment:
doc/source/programs/gdallocationinfo.rst to update with the new option

Reply:
done